runner/src/main/openapi/runner.yaml

```diff
@@ -674,6 +675,124 @@ components:
+      type: object
+      description: Request object for creating a new runner
+      properties:
+        name:
+          type: string
+          description: the Runner name
+        description:
+          type: string
+          description: the Runner description
+        tags:
+          type: array
+          description: the list of tags
+          items:
+            type: string
+        solutionId:
+          type: string
+          readOnly: true
+          description: the Solution Id associated with this Runner
+        parentId:
+          type: string
+          description: the Runner parent id
+        runTemplateId:
+          type: string
+          description: the Solution Run Template Id associated with this Runner
+        datasetList:
+          type: array
+          description: the list of Dataset Id associated to this Runner Run Template
+          items:
+            type: string
+        runSizing:
+          description: definition of resources needed for the runner run
+          $ref: "#/components/schemas/RunnerResourceSizing"
+        parametersValues:
+          type: array
+          description: the list of Solution Run Template parameters values
+          items:
+            $ref: '#/components/schemas/RunnerRunTemplateParameterValue'
+        ownerName:
+          type: string
+          readOnly: true
+          description: the name of the owner
+        solutionName:
+          type: string
+          readOnly: true
+          description: the Solution name
+        runTemplateName:
+          type: string
+          readOnly: true
+          description: the Solution Run Template name associated with this Runner
+        security:
+          $ref: '#/components/schemas/RunnerSecurity'
+      required:
+        - name
+        - solutionId
+        - runTemplateId
+        - ownerName
+
+    RunnerUpdateRequest:
+      type: object
+      description: Request object for updating a runner
+      properties:
+        name:
+          type: string
+          description: the Runner name
+        description:
+          type: string
+          description: the Runner description
+        tags:
+          type: array
+          description: the list of tags
+          items:
+            type: string
+        runTemplateId:
+          type: string
+          description: the Solution Run Template Id associated with this Runner
+        datasetList:
+          type: array
+          description: the list of Dataset Id associated to this Runner Run Template
+          items:
+            type: string
+        runSizing:
+          description: definition of resources needed for the runner run
+          $ref: "#/components/schemas/RunnerResourceSizing"
+        parametersValues:
+          type: array
+          description: the list of Solution Run Template parameters values
+          items:
+            $ref: '#/components/schemas/RunnerRunTemplateParameterValue'
+        ownerName:
```

Comment:
I think that we can remove `ownerName` property.
It's not used and there's already `ownerId` :)